mungegithub/mungers/blunderbuss.go

```diff
@@ -125,11 +126,25 @@ func getPotentialOwners(author string, feats *features.Features, files []*github
 			if owner == author {
 				continue
 			}
+			// Inside of a vendor directory, it is possible that
```

Comment:
I am on the fence. If @eparis does not feel strong, I would say start with `vendor/` and open an issue to potentially extend it globally.